src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    private String bulldozerName;
+
+    public Bulldozer(String bulldozerName) {
+        this.bulldozerName = bulldozerName;
+    }
+
+    public String getBulldozerName() {
+        return bulldozerName;
+    }
+
+    public void setBulldozerName(String bulldozerName) {
+        this.bulldozerName = bulldozerName;
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(bulldozerName + " started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(bulldozerName + " has stopped working");
+    }
+}
```

Comment:
```suggestion
public class Bulldozer extends Machine {

    @Override
    public void doWork() {
        System.out.println("Bulldozer started working");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer has stopped working");
    }
}
```
You don't need this field because a bulldozer will always be a bulldozer. Fix everywhere